### ISSUE TITLE
japscan.vip: ads / detection / breakages

### ISSUE THREAD

**Issue by tomtomnc1** [uBlockOrigin/uAssets#23806]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.japscan.lol/`

### Category

detection

### Description

a window asking to desable add blocker mask the view and the use

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.57.2
Firefox Mobile: 126
filterset (summary):
 network: 154565
 cosmetic: 61401
 scriptlet: 19779
 html: 1816
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 38840-113, 27m
  ublock-badware: 8785-0, 27m
  ublock-privacy: 921-2, 23m Δ
  ublock-unbreak: 2341-2, 27m
  ublock-quick-fixes: 157-15, 27m
  adguard-mobile: 9390-69, 27m
  easylist: 87322-228, 23m Δ
  easyprivacy: 50958-72, 23m Δ
  urlhaus-1: 13145-1, 27m
  plowe-0: 3726-1233, 27m
  FRA-0: 24451-507, 27m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 205 ms (selfie)
 maxAssetCacheWait: 62 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 19
 network:
  japscan.lol: 8
  bootstrapcdn.com: 2
  fadingsulphur.com: 2
  google.com: 3
  googletagmanager.com: 1
  ko-fi.com: 2
  statorkumyk.com: 1
 extended:
  ###mgid_iframe1
  ##.copinees
  ##div.container > [href]
  ##+js(no-setTimeout-if, detectImgLoad)
  ##+js(abort-current-script, document.addEventListener, /getCompu…
  ##+js(no-window-open-if)
  ##+js(remove-node-text, script, https://www.aozoiekopoaka.shop/)
```
</details>

**Comment by stephenhawk8054** [comment]
```adb
japscan.lol##+js(nostif, sessionStorage)
japscan.lol##.copineess
```

@Yuki2718 For Adguard French?

**Comment by stephenhawk8054** [comment]
Don't use multiple blockers at the same time. I can't reproduce the detection. Test again in a new browser profile with only uBO and its default settings.

The above filters are just for the ads on the page. I forgot to note that.

**Comment by tomtomnc1** [comment]
I dont use them at the same time. I deleted Adguard for only UBO, but its detected again. i didnt change the settings. And when i tryed Aguard i desactivated uBO.

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by stephenhawk8054** [comment]
Ok, I can reproduce it now on mobile. This might help

```adb
@@||japscan.lol^$ghide
```

**Comment by stephenhawk8054** [comment]
@Yuki2718 Oh sorry, the anti-adblock is not related to `ghide`, it's this one

```adb
japscan.lol##+js(acs, addEventListener, /blur-?inter/i)
```

**Comment by WhiskasChaton** [comment]
yesterday it was working with 
@@||japscan.lol^$ghide
japscan.lol##+js(acs, addEventListener, /blur-?inter/i)

but today the blur is back, even with 
japscan.lol##+js(acs, addEventListener, /blur-?inter/i)

**Comment by stephenhawk8054** [comment]
There's a new update in `Adguard French` list: https://github.com/AdguardTeam/AdguardFilters/commit/4bc3d4e21f650e95d2ac863129dda6c80c6aa20e

**Comment by tomtomnc1** [comment]
Just now the content is now blocked by UBO.

**Comment by stephenhawk8054** [comment]
Already fixed here: https://github.com/uBlockOrigin/uAssets/commit/ae5566ed74ee7bb6ce3bf3f5522894c02c48d7be

**Comment by Maddos75** [comment]
Is it fixed? I'm still having the issue.

**Comment by RedSkull-666** [comment]
Detect uBlock Origin again

**Comment by RedSkull-666** [comment]
Detect uBlock Origin again

<details>
<summary>Screenshot(s)</summary>



![image](https://github.com/uBlockOrigin/uAssets/assets/60255796/f7a9825b-3d68-4b48-ad0d-c93271fa1d93)




</details>

**Comment by RedSkull-666** [comment]
Detect uBlock Origin again

**Comment by D4niloMR** [comment]
`||japscan.lol/*=$popup` is breaking open link in new tab.

![image](https://github.com/uBlockOrigin/uAssets/assets/70459964/3eb8daa4-84ac-456d-a3ce-61bbdf38ab4f)

**Comment by avotoko** [comment]
works for me.
```
! avoid detection
japscan.lol##+js(set, oawOA, noopFunc)
```

**Comment by D4niloMR** [comment]
@smed79 For the "copines AI" ad in the home page:

`japscan.lol###main > .card:has(a[href*="/manga/?type="][href*="&content="][target="_blank"])`

**Comment by avotoko** [comment]
The site changed the targeted code.
```
japscan.lol#@#+js(aeld, load, /\('id'\)|setTimeout/)
japscan.lol##+js(aeld, load, /\("id"\)|setTimeout/)
```

**Comment by WhiskasChaton** [comment]
blur is up again...

**Comment by avotoko** [comment]
works for me
```
! avoid detection
japscan.lol###blur-inter:remove()
```

**Comment by avotoko** [comment]
site updated
```
! avoid detection
japscan.lol#@##blur-inter:remove()
japscan.lol##+js(set, XAoQp, noopFunc)
```

**Comment by D4niloMR** [comment]
[Click here](https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=FRA-0) to update the regional list and test again.

**Comment by avotoko** [comment]
Sorry, I neglected to update the french list. It is working.

**Comment by RedSkull-666** [comment]
Detect uBlock Origin again

**Comment by justinbridouille** [comment]
detect again,
and what should  ##+js(aopr, getComputedStyle) do anyway ??

**Comment by RedSkull-666** [comment]
Detect uBlock Origin again

**Comment by gorillaze05** [comment]
Hi, it detects Ublock again.
Thanks

**Comment by gorillaze05** [comment]
Hi, Ublock is detected once again.
Thanks

**Comment by crakwars** [comment]
> Hi, Ublock is detected once again. Thanks

Hi, it's the same for me, 
in the meantime I've noticed that by deactivating the ublock list :
 ublock filters-quickfixes , the addblock killer disappears.

If it helps^^

**Comment by gorillaze05** [comment]
Hi Crakwars, 
Thanks for the tip but it does not work for me!

**Comment by justinbridouille** [comment]
blocking the blur-inter is often detected, this shouls work as well
japscan.lol###blur-inter:style(visibility: hidden !important)

**Comment by crakwars** [comment]
> blocking the blur-inter is often detected, this should work as well japscan.lol###blur-inter:style(visibility: hidden !important)

don't work for me :/
today, many of ublock's lists seem to pop antiadblock on japscan, each launched as the only active list:
ublock filters ads
ublock filter quick fixes
easylist
Adguard french...
I haven't tried everything but it's sensitive today

**Comment by gorillaze05** [comment]
Hi, Ublock is detected once again.
Thanks

**Comment by justinbridouille** [comment]
It's been too long since I used Github, I don't remember if i need to fork

**Comment by Ever4engel** [comment]
Hi, it detects Ublock again.
Thanks

**Comment by gorillaze05** [comment]
Hi, Ublock is detected again
Thanks

**Comment by avotoko** [comment]
works for me
```
japscan.lol#@#+js(abort-on-stack-trace, getComputedStyle, _0x)
japscan.lol##+js(set, playGame, "")
japscan.lol##+js(set, vDhAo, true)
```

**Comment by D4niloMR** [comment]
@gorhill `japscan.lol#@#+js(aeld, load, /function.{1\,3000}$/)` (in custom filters) is not working on Firefox. Creating an exception in the logger doesn't work either.

<details><summary>Screenshot</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/70459964/08bc3ba9-4085-4af3-853b-187fcf781b1a)

</details>

**Comment by gorhill** [comment]
I don't understand what is going on. I can see scriptlet filters in the logger which are not normalized, while such filters are normalized at compile time. Trying to investigate the issue, I disabled all the uBO list so that I could step in the code after adding a single custom scriptlet filter to confirm the normalization code is called. Then I re-enabled all uBO filters and now I see all the scriptlet filters are normalized as expected, thus the exception filter works. I am clueless about what is happening.

**Comment by Ever4engel** [comment]
Hi, Ublock is detected again
Thanks!

**Comment by Ever4engel** [comment]
Hi, Ublock is detected again
Thanks

**Comment by E1ght52** [comment]
Hi i am agry with Ever4engel Ublock detected

**Comment by gorillaze05** [comment]
Hi it worked this morning for a couple of hours and it is down again.
Thanks

**Comment by Ever4engel** [comment]
It seems that the detection issue happens on firefox from time to time.
when you refresh, sometimes you don't get the pop-up and sometimes it's displayed.

**Comment by SeamlessRelaxation** [comment]
for user filters (temporarily):
```adblock
japscan.lol#@#+js()
@@||japscan.lol^$ghide
```

**Comment by justinbridouille** [comment]
just 
japscan.lol#@#+js(rmnt, script, /while \(!!\[\]\)|String\(\)\.fromCharCode|\.join\(\w+\)\.split\(\w+\)|"undefined"|_\$|\[\d+\][^=]*==|\.join|\.substr|\.charAt/)

**Comment by Ever4engel** [comment]
Side note for developers here: this website's main developer is monitoring this ticket constantly to update his code asap.

**Comment by justinbridouille** [comment]
so let's see who is the most tenacious

**Comment by WhiskasChaton** [comment]
Detect uBlock again

**Comment by Yuki2718** [comment]
> Detect uBlock again

I can't reproduce.

**Comment by WhiskasChaton** [comment]
Worked for few minutes then detect again. Looks like the website admin hardly react to any change :/ Have you solutions for this ?

**Comment by Ever4engel** [comment]
He even did something yesterday causing excessive usage of ram when you use ublock origin for firefox users.

**Comment by Yuki2718** [comment]
> He even did something yesterday causing excessive usage of ram when you use ublock origin for firefox users.

That may rather be due to a change in French list yesterday.

**Comment by D4niloMR** [comment]
@gorhill `japscan.lol##+js(acs, Array.prototype.join)` is preventing scriptlet logging of another scriptlets, for example: 

`japscan.lol##+js(acs, document.querySelector)`

Once I make an exception for it, I can see the logging of the scriptlet above.


<details><summary>Console output</summary>

```
Uncaught InternalError: too much recursion
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
    get blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:101
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
    get blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:101
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
2429f0b4-4319-40ca-ae18-cd1c993ff83f:276:26
Uncaught InternalError: too much recursion
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
    get blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:101
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
    get blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:101
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
2429f0b4-4319-40ca-ae18-cd1c993ff83f:276:26
Uncaught InternalError: too much recursion
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
    get blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:101
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
    get blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:101
    sendToLogger blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:276
    uboLog blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:188
    validate blob:https://www.japscan.lol/2429f0b4-4319-40ca-ae18-cd1c993ff83f:89
2429f0b4-4319-40ca-ae18-cd1c993ff83f:276:26
Uncaught ReferenceError: Countdown is not defined
    <anonymous> https://www.japscan.lol/:79
www.japscan.lol:79:44
Uncaught ReferenceError: jQuery is not defined
    <anonymous> https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.js:17
    <anonymous> https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.js:17
slick.min.js:17:152
Uncaught TypeError: can't access property "fn", t is undefined
    P https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/js/bootstrap.min.js:6
    <anonymous> https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/js/bootstrap.min.js:6
    <anonymous> https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/js/bootstrap.min.js:6
    <anonymous> https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/js/bootstrap.min.js:6
bootstrap.min.js:6:2080
Uncaught ReferenceError: QnEOPD is not defined
    jso$ft$giden$QnEOPD https://www.japscan.lol/:17
    af1428276 https://www.japscan.lol/:17
    <anonymous> https://www.japscan.lol/:63394
www.japscan.lol:17:110

```

</details> 

---

Additionally, (and a bit off topic), can the verbose mode be enabled by default in the dev build, or an advanced setting to keep verbose mode persistenly enabled?

**Comment by garry-ut99** [comment]
> Console output

Also the error message varies, depending on a browser: [LINK](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Too_much_recursion)
"Uncaught InternalError: too much recursion (Firefox)"
"Uncaught RangeError: Maximum call stack size exceeded (Chrome)"

<details>
<summary>Chromium console output</summary>

```
VM36:101 Uncaught RangeError: Maximum call stack size exceeded
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
VM36:49 Uncaught RangeError: Maximum call stack size exceeded
    at getScriptText (<anonymous>:49:27)
    at validate (<anonymous>:82:28)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
VM36:101 Uncaught RangeError: Maximum call stack size exceeded
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
VM36:101 Uncaught RangeError: Maximum call stack size exceeded
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
VM36:100 Uncaught RangeError: Maximum call stack size exceeded
    at Array.get (<anonymous>:100:26)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)
    at Object.uboLog (<anonymous>:188:25)
    at validate (<anonymous>:93:14)
    at Array.get (<anonymous>:101:17)
    at safe.sendToLogger (<anonymous>:276:87)

```
</details>

> is preventing scriptlet logging of another scriptlets

The logic is following:
`abort-current-inline-script.js` depends on 
`abort-current-script-core.fn` (https://github.com/gorhill/uBlock/blob/a54d416143b89f26417393fde76de078dc0b1dc8/assets/resources/scriptlets.js#L1448-L1449) 
which depends on `safe-self.fn` (https://github.com/gorhill/uBlock/blob/a54d416143b89f26417393fde76de078dc0b1dc8/assets/resources/scriptlets.js#L364-L366)
which depends on: `args.join` (https://github.com/gorhill/uBlock/blob/a54d416143b89f26417393fde76de078dc0b1dc8/assets/resources/scriptlets.js#L173)
which is aborted / interferrenced by : `japscan.lol##+js(acs, Array.prototype.join)` 
and it causes "infinite recursion loop" described in the Mozilla site, which is aborted by a browser.

**Comment by Ever4engel** [comment]
Hey, Ublock origin got blocked again.
Funny enough a certain workaround still work like a charm

**Comment by Ever4engel** [comment]
UBorigin got blocked again.
side question:

when someone commit a fix like here 
![image](https://github.com/user-attachments/assets/01e4413a-1d2d-4a27-b562-0fe174d84af1)
how much time will the fix take to be pushed to the filter list?

from what i seen it take few hours usually.

**Comment by garry-ut99** [comment]
> side question, how much time will the fix take to be pushed to the filter list? from what i seen it take few hours usually.

About 5 hours (unless devs provide a special hot-fix link, like here: https://github.com/uBlockOrigin/uAssets/issues/23806#issuecomment-2166896502 , then it can be updated instantly once you click it):

- https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#auto-update-filter-lists
- https://github.com/uBlockOrigin/uBlock-issues/issues/3263#issuecomment-2148008070

**Comment by Ever4engel** [comment]
Blocked again. only few hours after it's live in the list.

can we obfuscate the workarround (fix)? not sure if gorhill gonna accept tho.

he'll make quite some extra time to figureout what the fix do exactly.

**Comment by Ever4engel** [comment]
Origin blocked again

**Comment by Ever4engel** [comment]
Origin blocked again. the developer fixed it even before the fix goes live.

**Comment by Ever4engel** [comment]
Ublock blockd. so ye he fix before it get pushed, he just read what is the fix and then go change his js.

**Comment by Ever4engel** [comment]
So ye, he is constantly monitoring this ticket asap @Yuki2718 or @D4niloMR PR a fix he just break the fix in his website.
So basically he patch his website to avoid the workaround before the fix gonna be shared in the live list.

let's jump to conclusions: we need to do those to not let him time to fix

1- heavily obfuscate the workaround (dunno if @gorhill gonna accept tho)
2- push the pr asap as emergency push update.

or just drop support for this website, but in this case japscan owner just won the battle.

**Comment by D4niloMR** [comment]
Obfuscation won't be implemented. Related discussion: https://github.com/uBlockOrigin/uBlock-issues/discussions/2990

**Comment by Ever4engel** [comment]
it got blocked again even before going live to list.


it's a lost battle i think, better to close this and chage it to won't fix.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
it got blocked again.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
Dunno what he is doing to be honest, from two days ago, sometimes the website detect ublock origin and sometimes nope... 
but from an hour or so the website spoting directly ublock origin and giving the popup addblock detected.

**Comment by gorillaze05** [comment]
origin blocked

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked and infinite loading on the page "verifying you are human"

**Comment by Ever4engel** [comment]
Seem he modified his approach.
the adblock popup now occur in the reader and not in the main page anymore.

**Comment by Yuki2718** [comment]
> Seem he modified his approach. the adblock popup now occur in the reader and not in the main page anymore. 

I'm  aware of the change.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
Blocked.

**Comment by RedSkull-666** [comment]
Detect uBlock Origin again

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by RedSkull-666** [comment]
detected again

**Comment by Ever4engel** [comment]
blocked.

**Comment by Ever4engel** [comment]
Right now the pop up isn't showing in the reader. but seem images don't load anymore.

**Comment by gorillaze05** [comment]
detected again

**Comment by Yuki2718** [comment]
> detected again

Can't reproduce with the latest filters, please update Quick fix and AdGuard Français.

**Comment by Ever4engel** [comment]
it's blocked on my side, maybe adguard Français being late.
i can see your changes in quick fixes list.

**Comment by gorillaze05** [comment]
Blocked again

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
Ublock origin blocked by another fix.

**Comment by Ever4engel** [comment]
Origin blocked.

he came back to block on homepage again.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Zaitooo** [comment]
origin blocked

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by Ever4engel** [comment]
origin blocked.

He started again to fix his code asap something new get pushed here.

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
there a portion of ads that need to be added to the list i think

**Comment by gorillaze05** [comment]
origin blocked

**Comment by Yuki2718** [comment]
Fixed 10 minutes ago

**Comment by gorillaze05** [comment]
blocked again

**Comment by Ever4engel** [comment]
this is a hardcoded attachement in the main home page (the link is litterally an ad link).



it is possible to remove this with cosmetique lists?

i mean i can remove it locally but it would better to remove it by lists.

**Comment by Yuki2718** [comment]
I've been targetting that AI Girlfrined button, but they make my filter to also trigger the anti-adblock.

**Comment by gorillaze05** [comment]
Origin blocked again

**Comment by gorillaze05** [comment]
Thanks but it is blocked again

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Dude is on crusuade...

Origin blocked.

Well he's that dedicated cuz he have an ad-free plan for his paid readers, they pay him an amount of money and he give them acsses to a special page(?). anyway he use it so it don't break the website i guess because imagine each time webpage is down while he patch those workarounds.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Yuki2718** [comment]
> Origin blocked.

I can't reproduce, can you check with Quick force-updated by holding Shift when clicking the clock icon?

**Comment by Ever4engel** [comment]
> > Origin blocked.
> 
> I can't reproduce, can you check with Quick force-updated by holding Shift when clicking the clock icon?

did that, forced the update (with shift) for both quick fixes and adguard français list. the pop-up shown.

**Comment by Yuki2718** [comment]
I now get this

https://github.com/user-attachments/assets/a07a08a6-d57a-4231-880c-8f412bf4145f)

> Edit: side question, why your fix need to be push on both those lists?

Because I'm a maintainer of AdGuardFilters too. It's updated very quickly in AdGuard extensions with diff-update. I already discussed about this in internal discussion and gorhill promised it will eventually be taken care of, though not a trivial fix.

**Comment by Ever4engel** [comment]
now they're up again.

https://github.com/user-attachments/assets/9b755d79-bc26-4100-b59b-19cbe47f5751)

i see (for the question).

**Comment by gorillaze05** [comment]
origin blocked

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Yuki2718** [comment]
> Origin blocked

Can't reproduce with the latest filters.

**Comment by gorillaze05** [comment]
> > Origin blocked
> 
> Can't reproduce with the latest filters.
Ok maybe I am missing something, I am updating the filters (by clicking on the clocks) but it is still detected

**Comment by Yuki2718** [comment]
Alright, I can reproduce only on Firefox.

**Comment by Ever4engel** [comment]
right now on firefox, work just fine.

Edit: home page no popups
but on the reader side there a popup

**Comment by garry-ut99** [comment]
Appears again on main page and on reader pages, on Firefox (but not on Chromium).

**Comment by 055110** [comment]
Origin blocked again? God this guy is obsessed over his ad revenue or something.

**Comment by Ever4engel** [comment]
> Origin blocked again? God this guy is obsessed over his ad revenue or something.

the the worst, he just comes here randomly, see if there a new patch in the code, then while it getting few hours to get approved, he just break that new patch.

just to kill yiki's motivation.

**Comment by garry-ut99** [comment]
Appears again, this time on both Firefox and Chromium for me.

**Comment by Yuki2718** [comment]
> just to kill yiki's motivation.

Motivation has never mattered to me. It's not the first or second time I played a cat-and-mouse game for months. Fine to keep for years if the site owner wants.

**Comment by gorillaze05** [comment]
Origin detected again

**Comment by gorillaze05** [comment]
Origin detected

**Comment by gorillaze05** [comment]
> Origin detected

I think he broke his website like last time

**Comment by Ever4engel** [comment]
guys you need to (usually) make sure the rule is Pushed to your local version (usually after ~4/5 hours).

right now firfox work just fine.
tried on edge, ye it's detected.

**Comment by Yuki2718** [comment]
The latest fix is still working both on Firefox and Chromium on my end as of writing now.

**Comment by Yuki2718** [comment]
> I think he broke his website like last time

Possibly they may remember IP addresses that consistently keep using blocker and show warning for them even without blocker?

**Comment by Ever4engel** [comment]
right now in Edge


and now also firefox is just blocked it seems.

**Comment by garry-ut99** [comment]
Has appeared again on both Firefox & Chromium.

**Comment by S3BSama** [comment]
> > I think he broke his website like last time
> 
> Possibly they may remember IP addresses that consistently keep using blocker and show warning for them even without blocker?

I  tried to use a vpn (non french IP) + privacy mode in Firefox and the ads are blocked! (WIthout the VPN uBlock is "detected")

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by gorillaze05** [comment]
origin blocked

**Comment by Ever4engel** [comment]
origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
a question yuki can we add your new rules to our personal filters in ubo intill it get pushed to wild ?

**Comment by Yuki2718** [comment]
> a question yuki can we add your new rules to our personal filters in ubo intill it get pushed to wild ?

Sure, no need to wait for the update. Just be careful that the filter may then trigger anti-adb on their next update, so you have to not only add but also remove the previous one if it's removed or disabled with `@` in the commit.

**Comment by Ever4engel** [comment]
> > a question yuki can we add your new rules to our personal filters in ubo intill it get pushed to wild ?
> 
> Sure, no need to wait for the update. Just be careful that the filter may then trigger anti-adb on their next update, so you have to not only add but also remove the previous one if it's removed or disabled with `@` in the commit.

Okey gotcha so basically i just need to copy pasta last revision here when you push and update
![image](https://github.com/user-attachments/assets/37261b95-eec4-402d-94af-bf1e6138b4a3)

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
@Yuki2718 i did added japscan filters manually 

https://github.com/user-attachments/assets/8dcce8d8-3c42-4340-adaa-73f7b84a716f)

this is the last revision you made.
still it's blocked. what mistake i did ?

**Comment by Yuki2718** [comment]
@Ever4engel They already changed.

**Comment by Ever4engel** [comment]
> @Ever4engel They already changed.

i see, so basically after an hour they already changed, that crazy.

------------------------
Edit: less than two minutes after i commented, i checked the website it was just fine.
But another two minutes later he patched his webside.
seem he activated this 
https://github.com/user-attachments/assets/165815ae-3a09-4109-84e8-c4d18a6caa7f)
with his dev account.

Anyway, **Origin blocked.**

**Comment by cijimac783** [comment]
origin blocked

**Comment by cijimac783** [comment]
The owner of the japscan site regularly comes to the Ublock Origin GitHub to see if a modification has been made, it's a waste of time.

**Comment by Yuki2718** [comment]
> seem he activated this

A few weeks ago, @stephenhawk8054 made some "silent" commits among mine, nevertheless the site owner responded. Maybe script or somthing, IDK.

**Comment by Ever4engel** [comment]
this time you somehow pushed asap to wild.
good, it work on my side.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
> You are bypassing the anti-adblocking but not removing the ads on the site itself. Issue occurs on the following page: `https://www.japscan.lol/manga/the-hated-black-mage-returns/`

Ye developer added another manual ad placeholder. yuki should block it in due time.

the ad are here 
(https://github.com/user-attachments/assets/5485436d-0d6e-43b8-a2a4-61980c4e2fd6)
(https://github.com/user-attachments/assets/80f6ebf5-34d6-49c6-a338-deba9011f5ae)

----------

Edit: **Origin blocked.**

**Comment by Yuki2718** [comment]
> You are bypassing the anti-adblocking but not removing the ads on the site itself. Issue occurs on the following page: 

The fact is, if I bypass anti-adb then the site owner tries to exploit this to show ads. It's trivail to block all ads if I can ignore the anti-adb, happy to hear if you have a good idea.

**Comment by Yuki2718** [comment]
> Have you tried _replacing_ the ad image with nothing?

It doesn't matter when they watch us. Redirecting to neutered resources, removing, css tricks - all detected.

**Comment by Ever4engel** [comment]
Right now to do a fair mouse cat play with him:

1- Yuki push to upstream his code directly.
2- don't post the fix here (yuki) intill he push all websites fixes to upstream.
3- yuki get the ability to push upstream instant for each fix he think it need to be pushed urgently.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Han-O-Neam** [comment]
Hi there.
Even UBO disabled, it is detected and the site is blocked.

**Comment by Yuki2718** [comment]
@Han-O-Neam Maybe your IP is blacklisted by them.

**Comment by Han-O-Neam** [comment]
> @Han-O-Neam Maybe your IP is blacklisted by them.

Hmm. Seems to be temporary ;
Maybe it was due to multiple successive bad requests that drove me to be blacklisted for a while (which meets your comment).
I hope there will be soon an update of rules to be able to run UBO again, as this site is really overwhelmed of intrusive's ads.
I tried to begin to analyse the js and/or find which script to block but without success at this time.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
seem somthing is broken while it mitigate cloudflare chalenge test in reader.

without Ub0 it work just fine.

**Comment by Yuki2718** [comment]
> seem somthing is broken while it mitigate cloudflare chalenge test in reader.
> 
> without Ub0 it work just fine.

Yeah, my fault.

**Comment by Ever4engel** [comment]
Origin blocked in reader part.
on home page not yet.

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by white12130** [comment]
Origin blocked

**Comment by Yuki2718** [comment]
> Origin blocked

can't reproduce with the latest filters.

**Comment by white12130** [comment]
blocked again

**Comment by gorillaze05** [comment]
Blocked

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by white12130** [comment]
Origin blocked

**Comment by gorillaze05** [comment]
Origin blocked

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by white12130** [comment]
Origin blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by white12130** [comment]
origin blocked

**Comment by Ever4engel** [comment]
work just fine, update your filters.

**Comment by Ever4engel** [comment]
ub0 blocked.

**Comment by white12130** [comment]
Origin Blocked     https://www.japscan.lol/lecture-en-ligne/undead-unluck/223/

**Comment by white12130** [comment]
not blocked on homepage but blocked on chapter player      https://www.japscan.lol/lecture-en-ligne/undead-unluck/223/

**Comment by white12130** [comment]
blocked again

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Yuki2718** [comment]
> Origin blocked.

Fixed a minute before your comment.

**Comment by gorillaze05** [comment]
Blocked again

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by white12130** [comment]
Origin Blocked https://www.japscan.lol/lecture-en-ligne/undead-unluck/223/

**Comment by gorillaze05** [comment]
Blocked

**Comment by Yuki2718** [comment]
> but the scans are not showing

For me the same without blocker. It seems the site is broken now.

**Comment by white12130** [comment]
same problem for me. the chapters are not displayed even without blocking ads

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by white12130** [comment]
Origin Blocked.

**Comment by white12130** [comment]
Origin Blocked    https://www.japscan.lol/lecture-en-ligne/undead-unluck/223/

**Comment by Ever4engel** [comment]
Guys it's better to let only one post saying ''Origin Blocked." to let this ticket as clean as possible, and avoid unnecessary spam.
Yuki and other contributors have lives, posting multiple messages wont speed up the process. thank you all.

**Comment by white12130** [comment]
blocked again

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by white12130** [comment]
Still blocked

**Comment by white12130** [comment]
still stuck. thanks to Yuki2718 for your efforts

**Comment by white12130** [comment]
still blocked

**Comment by gorillaze05** [comment]
Blocked

**Comment by white12130** [comment]
Blocked

**Comment by white12130** [comment]
still blocked

**Comment by white12130** [comment]
Blocked

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by white12130** [comment]
Blocked .

**Comment by Ever4engel** [comment]
> Blocked .

nothing to do with the website. website is just down (invalid certificate).

**Comment by white12130** [comment]
Blocked

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by white12130** [comment]
Blocked

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by gorillaze05** [comment]
Blocked

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by white12130** [comment]
Blocked

**Comment by Yuki2718** [comment]
> Blocked

can't reproduce

**Comment by white12130** [comment]
it's weird I updated the Quick fixes filters and I still have the blocked message. I'm on chrome

**Comment by white12130** [comment]
blocked again

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by Ever4engel** [comment]
Origin Blocked.

**Comment by Yuki2718** [comment]
Guys, test adding `japscan.lol##+js(aeld, DOMContentLoaded, nCpC)` to My filters. It doesn't work for the main page, but for that you can block inline-script if you accept a little breakage. Let's see if the owner is watching comments here or not.

(https://github.com/user-attachments/assets/17a15507-d046-454a-a148-be8ffcd9000a)

(alternatively `||japscan.lol^$inline-script`)

**Comment by gorillaze05** [comment]
Maybe I didn't do things correctly but it is still blocked on my side

**Comment by Yuki2718** [comment]
@gorillaze05 Can you check if you have the latest filters? Click the eye icon next to uBlock filters – Quick fixes.

**Comment by Ever4engel** [comment]
@Yuki2718 it work just fine right now, i just added the cosmetic filter you asked to add and forced the update for the requested filter.

-------------------------------

Also i tried this:
(https://github.com/user-attachments/assets/7143eb98-d9fe-4180-b871-f00dc18c0751)

**+**


(just as tests)

and the website work just fine (no popup), but no image can load, also can't get the cloudflare challenge.

and yes the dude is watching this ticket 24/7.

-------------------------

there something i seen, when i go on the cloudflare challenge page i get a blob file dl'ing to my pc 
here the link https://goldilocks-free.research.cloudflare.com/block3.img?requestid=8d62ce446c0e28262

**Comment by Yuki2718** [comment]
> Also i tried this:

You don't have to keep them, but test on their next counter. Yes, blocking inline-script breaks images.

**Comment by Ever4engel** [comment]
oh ok so, when he block again the workaround i see if your filters still work
japscan.lol##+js(aeld, DOMContentLoaded, nCpC)
and ||japscan.lol^$inline-script

**Comment by Yuki2718** [comment]
If he is watching comments, `japscan.lol##+js(aeld, DOMContentLoaded, nCpC)` won't work. I wonder if he only runs script to check or only watches commits.

**Comment by Ever4engel** [comment]
Origin Blocked.
Edit: in home page, seems, when i press refresh button (in the popup), the adblock popup don't show again.



Edit2: he read this message and fixed that issue.

**Comment by Yuki2718** [comment]
Yep, he chose `nBpD`!

**Comment by white12130** [comment]
blocked again .

**Comment by gorillaze05** [comment]
blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by Ever4engel** [comment]
Origin Blocked.
home page just fine no popup.
reader: blocked.

**Comment by gorillaze05** [comment]
Blocked

**Comment by gorillaze05** [comment]
Blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by white12130** [comment]
Blocked

**Comment by white12130** [comment]
Blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by gorillaze05** [comment]
Blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by gorillaze05** [comment]
blocked

**Comment by gorillaze05** [comment]
Blocked

**Comment by gorillaze05** [comment]
Blocked

**Comment by gorillaze05** [comment]
Now using IP detection to put the wall even without adblocker

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by Ever4engel** [comment]
Origin blocked.

**Comment by RedSkull-666** [comment]
Hello, still detected and blocked

**Comment by Yuki2718** [comment]
> Adblock keeps detected. I tested with most adblocks include ublock origin (one by one), on all browsers, same result.

Use other sites.

**Comment by Ever4engel** [comment]
japscan seem down.

**Comment by gut1414** [comment]
It keep exists and his adblock detection message.

**Comment by Ever4engel** [comment]
japscan ip banned some countries. seem he try to narrow down the location of anti adblock contributors location.

**Comment by Ever4engel** [comment]
anti adblock detected on all browsers.

**Comment by Ever4engel** [comment]
Blocked again.

**Comment by Ever4engel** [comment]
Blocked.

**Comment by Ever4engel** [comment]
anti adblock script work fine, but infinite cloudflare death loop bug happening.
edit: he blocked again ub0

**Comment by Ever4engel** [comment]
domain name change to https://www.japscan.si

**Comment by Ever4engel** [comment]
i would recommand to change the title to .si to avoid any confusion.

**Comment by Ever4engel** [comment]
Ub0 detected

**Comment by Ever4engel** [comment]
Ub0 detected

**Comment by Ever4engel** [comment]
Ub0 detected

**Comment by Ever4engel** [comment]
Ub0 detected

i also feel he wrote a script to auto rotate/change how the antiadlocker works from here and then it somehwo update on his end.

**Comment by Ever4engel** [comment]
Ubo blocked on reader.

**Comment by Ever4engel** [comment]
Ubo blocked

**Comment by Ever4engel** [comment]
Ubo blocked

**Comment by Ever4engel** [comment]
Ubo blocked.

**Comment by Ever4engel** [comment]
Ubo blocked.

**Comment by Ever4engel** [comment]
Ubo blocked.

**Comment by Ever4engel** [comment]
Ubo blocked.

**Comment by Ever4engel** [comment]
noticed a potential bug.
i use firefox, default setings + ublock with default lists = antiadblock detected.
i use any chromium browser with actual lists (your last update) = anti adblock script not detected.

i felt like it needed a report here.
(i used a new portable firefox version with a raw profile + ubo with default lists just to make sure)

**Comment by Ever4engel** [comment]
Your workaround fixed my issue on firefox.

Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by ghost** [comment]
uBO detected, again.

**Comment by Yuki2718** [comment]
I can't reproduce.

**Comment by ghost** [comment]
Oops, I had the AdGuard French filters off, now with them on it did disappear. My bad.

**Comment by Ever4engel** [comment]
Ub0 blocked.
it got blocked just now.

**Comment by ghost** [comment]
Can confirm too.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by ghost** [comment]
Can't reproduce although it does say to stop the adblock if you browse through a manga.

**Comment by Yuki2718** [comment]
I can't reproduce as of writing.

**Comment by Ever4engel** [comment]
> I can't reproduce as of writing.

it was definitely blocked, he reverted to the previous rulesets (so now the script works just fine).

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by ghost** [comment]
Can confirm it got blocked.

**Comment by Ever4engel** [comment]
he reverted back to the previous valid rulesets again (dunno why). so basically now Ub0 isn't blocked anymore.

Edit: I just noticed something, on cloudflare challange, after you comptete it, the browser don't refresh the page automatically anymore and you remain stuck in challenge page, you need to F5 manually to pass the page.

**Comment by Ever4engel** [comment]
Ub0 blocked as of writing.

**Comment by Ever4engel** [comment]
Ub0 blocked in firefox, works pretty good on chromium browsers.

**Comment by Ever4engel** [comment]
Ub0 blocked

**Comment by Ever4engel** [comment]
Ub0 blocked in firefox, works pretty good on chromium browsers.

**Comment by Ever4engel** [comment]
same issue on firefox, the script don't work. all chromium browsers works fine and as expected.

**Comment by Ever4engel** [comment]
Ub0 blocked after a week, that's a record.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
this site migrated to https://www.japscan.vip
as time of writing not blocked yet.

**Comment by ghost** [comment]
Not sure if it's self-promotion or an ad, but it's at the bottom of the page while you read a manga.

**Comment by Yuki2718** [comment]
> Not sure if it's self-promotion or an ad, but it's at the bottom of the page while you read a manga.

I can't reproduce.

**Comment by ghost** [comment]
It's on mobile.

**Comment by ghost** [comment]
They are now hiding the pages of a comic if uBO is on. With uBO off they reappear

<details>

<img width="1919" height="1020" alt="Image" src="https://github.com/user-attachments/assets/3112c846-f516-46b5-845b-a3680506df2f" />
<img width="1919" height="1016" alt="Image" src="https://github.com/user-attachments/assets/3cd576e0-3fe5-486b-a26f-d08243497fd5" />

</details>

**Comment by ghost** [comment]
I force-updated the quick fixes with `Shift` key + click on the clock icon list but the breakage persists:
https://github.com/user-attachments/assets/247765a3-1eff-4380-85a0-eb4de4f81f31" />

**Comment by Ever4engel** [comment]
congrats yuki you won the war. i think this site changed their strat, they let it as is and they they randomly update their site once or twice a week/month.

**Comment by Yuki2718** [comment]
Will still keep an eye on the site, and merge rules with AdGuard Français (except for trusted filters) if everything goes fine.

**Comment by Ever4engel** [comment]
updated adguard français just now 


Ub0 blocked again @Yuki2718

**Comment by Yuki2718** [comment]
All right, we were too quick to judge.

**Comment by Ever4engel** [comment]
> All right, we were too quick to judge.

he will never give up, he just lost motivation/faith to fix the site everyday asap you push a fix.

so i really think the best thing to do is: let this ongoing, it'll send him a msg we're watching you.

**Comment by ghost** [comment]
uBO detected again with AG French filters on.

**Comment by Ever4engel** [comment]
uBO detected

**Comment by Yuki2718** [comment]
> uBO detected

Seems no longer the case

**Comment by ghost** [comment]
uBO detected with AG French filters on.

**Comment by Ever4engel** [comment]
uBO detected

**Comment by Ever4engel** [comment]
uBO detected

**Comment by Ever4engel** [comment]
uBO detected

**Comment by Ever4engel** [comment]
there a little cosmetic issue dunno if the report would be here or nah.



there a big empty space here (it is the ad placeholder), no ads but the space still like that.

ps: if it is unrelated to this section feel free to delete.

**Comment by ghost** [comment]
It's an ad leftover.

**Comment by Yuki2718** [comment]
I'm aware of that, but ignored. No worth hiding (= giving the site another way for detection).

**Comment by Ever4engel** [comment]
uBO detected.

**Comment by ghost** [comment]
Looks like they counter it a lot more faster than before.

**Comment by ghost** [comment]
uBO detected with AG French filters on.

**Comment by ghost** [comment]
Detection, again

**Comment by ghost** [comment]
Detection again

**Comment by ghost** [comment]
They moved really quick, they seem to counter uBO again even after force-updating the quick fixes list.

**Comment by ghost** [comment]
Detection, again
(in case it feels like I spam or something, tell me and I can simply edit my latest post)

**Comment by ghost** [comment]
uBO detected

**Comment by ghost** [comment]
uBO detected

**Comment by Ever4engel** [comment]
uBO detected.

**Comment by ghost** [comment]
Detection again

**Comment by ghost** [comment]
Detection again

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
@Yuki2718 dude changed his strategy, now he don't bother patch the stie, he just wait when you close this ticket, he modifies few lines to block it again.

Ob0 blocked.

Edit: basically while this ticket is open he wont change the site like what he used to be cuz he know it's a lost battle with your dedication.

**Comment by Ever4engel** [comment]
<https://github.com/user-attachments/assets/9a20b06d-10a9-4836-aabe-57bed3599a02" />

this killed me  XD

Ub0 blocked.

**Comment by Yuki2718** [comment]
Just report here regardless of open/close status and I'll counter whatever times.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by Ever4engel** [comment]
Ub0 blocked.

**Comment by ghost** [comment]
Detection again.

<details>
<img width="1771" height="862" alt="Image" src="https://github.com/user-attachments/assets/bcc65af9-ac22-494b-a36f-dd894f7c838e" />
<img width="1350" height="484" alt="Image" src="https://github.com/user-attachments/assets/882f8912-2a32-4402-a1e1-c7dab1ff778c" />
</details>

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

ex: https://www.japscan.vip/manga/usogui/227/1.html

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader again.

**Comment by ghost** [comment]
Reader again blocked if uBO is on.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Blocked in the reader, again

**Comment by Ever4engel** [comment]
it work just right on my end (firefox), either he reverted to the old blocked code or it affect only chromium browsers.
force refresh quick fixes list in ub0.

**Comment by ghost** [comment]
It works now on my side as well.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
And again blocked.

<details>

![Image](https://github.com/user-attachments/assets/593cfd32-ea04-4082-8803-2e8a09d39937)
![Image](https://github.com/user-attachments/assets/089bfdb7-e79d-499b-9499-0c582aac0aed)

</details>

**Comment by ghost** [comment]
Blocked in the reader, again

**Comment by Yuki2718** [comment]
Can’t reproduce for now

**Comment by pilis1** [comment]
Still blocked in the reader:

![Image](https://github.com/user-attachments/assets/28c75d36-83f9-4f15-8338-142791361f77)

I force updated the quick fixes list before.

**Comment by ghost** [comment]
Now not only it's hidden but also detection message is shown in the reader:

**Comment by Ever4engel** [comment]
@pilis1 when yuki checked it was really working, 
but now ye Ub0 blocked in reader.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Blocked again in the reader.

<details>

https://github.com/user-attachments/assets/cefde3e5-3745-47de-bc8f-ae6f12ab0146)
https://github.com/user-attachments/assets/e64e7d72-3b34-4186-9b4d-927e2a50cb0a)

</details>

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by geekx50** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Blocked in the reader, again

**Comment by Yuki2718** [comment]
Unable to reproduce

**Comment by ghost** [comment]
<details>

https://github.com/user-attachments/assets/852f9afc-099a-4bef-b88f-0907d57f1232
https://github.com/user-attachments/assets/99bac580-0926-4437-a67b-ab7eaa43d3a3

</details>

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Detection in the reader:
<details>

https://github.com/user-attachments/assets/b7232622-4364-44b2-a115-d0dd2c836c3c)
https://github.com/user-attachments/assets/afaf2af0-d89b-4541-86bb-4bd75724d5c9)

</details>

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Detection, again

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Detection, again

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
Detection, again

**Comment by ghost** [comment]
Detection, again

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by ghost** [comment]
It surprises me that those guys don't give up with countering.

**Comment by Ever4engel** [comment]
home page broken, i can't see the series 

<img width="1681" height="692" alt="Image" src="https://github.com/user-attachments/assets/ec39a435-a580-4ad9-a716-4ba8d86749b6" />

**Comment by ghost** [comment]
Can't reproduce.

**Comment by Ever4engel** [comment]
> Can't reproduce.

same issue on all browsers (chromium or Gecko)
when lists are updated to last upstream this issue occur in home page. with Ub0 disabled the home page load just fine.

also Ub0 blocked in the reader now.

**Comment by ghost** [comment]
Now I can reproduce as well.

**Comment by Ever4engel** [comment]
https://github.com/user-attachments/assets/1b21c73f-cb5b-4c35-84bd-2348fbe2a9cf

Same issue on home page.
(firefox)

**Comment by Ever4engel** [comment]
Ub0 blocked in the reader.

**Comment by HeiCo2** [comment]
Confirm, blocked again with Firefox

**Comment by ghost** [comment]
If you wait 2-3 seconds, the detection message appears also on the homepage.

**Comment by ghost** [comment]
Detection again

**Comment by ghost** [comment]
Detection, again:
"https://github.com/user-attachments/assets/68cdd0ba-a7a2-4562-8077-a0323c1fdf40" />

**Comment by ghost** [comment]
Detection, again

**Comment by ghost** [comment]
This looks like an ad, but the link doesn't work:
https://github.com/user-attachments/assets/58918a93-f86f-439b-8b0d-dfcfd3c1db3d" />

**Comment by ghost** [comment]
Also something weird, probably they are updating the site

Detection on `https://www.japscan.vip/manhwa/the-duchess-has-a-deathwish/62/1.html`
https://github.com/user-attachments/assets/8d1e76a1-5854-44e9-962c-8eb1b2ff62de" />

but not on `https://www.japscan.vip/manga/kingdom/864/`
https://github.com/user-attachments/assets/13b6e9d6-316f-474c-b93d-6dbce37e694c" />

**Comment by ghost** [comment]
Detection again

**Comment by Yuki2718** [comment]
Unable to reproduce

**Comment by ghost** [comment]
They are now hiding the mangas in the homepage when uBO is on:

https://github.com/user-attachments/assets/f4986dab-3dbd-4f58-ac31-cc3ab0af4a63
https://github.com/user-attachments/assets/8dcc98f3-2963-44fb-902f-fe46bdb53750

And the captcha is also broken when uBO is on, it continously loads:
https://github.com/user-attachments/assets/dadaea8f-bd52-4c9b-b204-8d6935d827df

On this link, the pages don't load:
`https://www.japscan.vip/manhua/catastrophic-necromancer/232/1.html`

https://github.com/user-attachments/assets/aecfe987-0767-48d9-898d-1d0bb0c6a385

**Comment by Yuki2718** [comment]
Unable to reproduce. Are you sure your filters are up to date?

> On this link, the pages don't load: `https://www.japscan.vip/manhua/catastrophic-necromancer/232/1.html`

Only this was reproduced.

**Comment by ghost** [comment]
The captcha continously loading can be reproduced, and the detection:

https://github.com/user-attachments/assets/32888636-1126-424a-a4e8-60c718b613a8" />
https://github.com/user-attachments/assets/f0a56aa7-b94c-4f96-9949-9103df88de50" />
https://github.com/user-attachments/assets/36f2747e-d1a2-4b83-9797-c113434a3d86" />

**Comment by Yuki2718** [comment]
Okay, now reproduced.

**Comment by ghost** [comment]
Detection again

**Comment by Ever4engel** [comment]
also sometimes cloudflare challenge buged

https://github.com/user-attachments/assets/09f1b08a-4bd3-42f2-b127-a78c7228c655" />

it stay blank like in the screenshot.

**Comment by Yuki2718** [comment]
> also sometimes cloudflare challenge buged
> 
> https://github.com/user-attachments/assets/09f1b08a-4bd3-42f2-b127-a78c7228c655" />
> 
> it stay blank like in the screenshot.

I can't reproduce. See if adding
```
japscan.*#@#div[id]:not(#TogglerNav,#bds,#best-value,#blare-inter,#blir-nter,#block-left,#block-right,#bookmarks,#chapters,#challenge-success-text,#content,#footer-text,#full-reader,#gnt_xmst,#https\:\/\/www\.japscan\.vip,#list_chapters,#main,#mangas,#minimize,#move,#nav_Header,#notice-message,#quit,#reader,#report-modal,#report-success,#results,#searchForm,#sidebar,#tns-inner,#toolbar,#verifying-msg,#windowbar,.cb-container,.loading-verifying,.main-content > div, body > div,[id^="collapse-"],[id^="d-img"],[id^="tab"],[id^="tns1"],[id^="top"],[id^="single-reader"]):remove()
```

fixes that.

**Comment by ghost** [comment]
The captcha works fine for me now. However, detection appeared again:

https://github.com/user-attachments/assets/2ec8e368-2baf-4695-a8f1-71c26bb16167" />

**Comment by Ever4engel** [comment]
https://github.com/user-attachments/assets/4b9a64f4-07b9-42a0-98c1-680d0a208c15" />

still same issue on my end even with that custom rule.
and ub0 detected.

**Comment by ghost** [comment]
Detection again.

**Comment by Yuki2718** [comment]
> 
> still same issue on my end even with that custom rule. and ub0 detected.

Does `japscan.*#@#+js()` work? This disable all the scriptlet rules on the site.

**Comment by ghost** [comment]
For me that filter worked, the captcha now can be passed.

However, on mobile at least, they hide the new chapters that appeared on the comics, plus detection:

<details>

https://github.com/user-attachments/assets/f7a91e02-7980-4d64-ae08-412853eee1ea)

https://github.com/user-attachments/assets/e47accb5-879e-4878-98a8-bfe1c5dc8d4f)
https://github.com/user-attachments/assets/634109ce-0d4e-453a-b35a-69b20cebfe42)

</details>

**Comment by Ever4engel** [comment]
> > still same issue on my end even with that custom rule. and ub0 detected.
> 
> Does `japscan.*#@#+js()` work? This disable all the scriptlet rules on the site.

`japscan.*#@#+js()` ye seem to work.

Ub0 blocked in the reader.

**Comment by Yuki2718** [comment]
I still can't reproduce the CF issue. Test
```
japscan.*#@#+js(trusted-suppress-native-method, HTMLElement.prototype.querySelectorAll, '"/.*\[[^imns].+\].*/"')
japscan.*#@#+js(trusted-suppress-native-method, Element.prototype.hasAttribute, '"/[\S\s]+/"', prevent, inlineScript)
japscan.*#@#+js(trusted-suppress-native-method, Document.prototype.getElementsByTagName, '"*"', prevent, inlineScript)
```

instead of `japscan.*#@#+js()`.

**Comment by ghost** [comment]
These filters are also good, no more captcha continously loading.

However, detection again:

https://github.com/user-attachments/assets/250a65ba-74ea-465e-b011-9f6b99b4709d)
https://github.com/user-attachments/assets/5a1458c7-cd57-4008-aa63-dee09b5dab8f)

**Comment by Ever4engel** [comment]
> I still can't reproduce the CF issue. Test
> 
> ```
> japscan.*#@#+js(trusted-suppress-native-method, HTMLElement.prototype.querySelectorAll, '"/.*\[[^imns].+\].*/"')
> japscan.*#@#+js(trusted-suppress-native-method, Element.prototype.hasAttribute, '"/[\S\s]+/"', prevent, inlineScript)
> japscan.*#@#+js(trusted-suppress-native-method, Document.prototype.getElementsByTagName, '"*"', prevent, inlineScript)
> ```
> 
> instead of `japscan.*#@#+js()`.

the old and new ruleset work perfectly now. i guess i need to mention i use firefox, just in case..





> These filters are also good, no more captcha continously loading.
> 
> However, detection again:
> 
> https://github.com/user-attachments/assets/250a65ba-74ea-465e-b011-9f6b99b4709d) https://github.com/user-attachments/assets/5a1458c7-cd57-4008-aa63-dee09b5dab8f)

you need to force update your list (quick fixes list), hold maj and press in the update button.

**Comment by ghost** [comment]
In vain, detection persists:

https://github.com/user-attachments/assets/d0fb4294-ef08-4560-b669-0a8aebab27f5
https://github.com/user-attachments/assets/db0a355f-d439-482f-a347-1ae7d657b778

The owners counter the filters super fast.

**Comment by Ever4engel** [comment]
the hole site (home page) is behind a cloudflare wall now down right now.
also pilis when i checked it was just okey, maybe he reverted his changes.

https://github.com/user-attachments/assets/cc68b9d8-c2f1-4d6d-8b19-854d3f584df2" />

**Comment by Yuki2718** [comment]
> the old and new ruleset work perfectly now. i guess i need to mention i use firefox, just in case..

Does the issue still happen if without the custom rules?

**Comment by Ever4engel** [comment]
> > the old and new ruleset work perfectly now. i guess i need to mention i use firefox, just in case..
> 
> Does the issue still happen if without the custom rules?

yes. that challenge goes blank.

https://github.com/user-attachments/assets/9d8a371b-cb8f-40dd-9297-7395a2ae1aca" />

when i pass the challenge in the reader he blocked again Ub0

**Comment by Yuki2718** [comment]
Looks like it's the latest filter causing the issue. I [disabled](https://github.com/uBlockOrigin/uAssets/commit/6500150bd10c0e976d424cf858d65c624759122e) that.

**Comment by ghost** [comment]
Detection, again

**Comment by ghost** [comment]
Detection, again

**Comment by Yuki2718** [comment]
Unable to reproduce right now

**Comment by Ever4engel** [comment]
https://github.com/user-attachments/assets/afecfefa-6c5d-4e58-bf2f-4084d71da4f0" />

firefox 147.0.3, Ub0 last version, last upstream lists. blocked in the reader.

**Comment by Ever4engel** [comment]
noticed those custom rules arn't needed for chromium browsers (i tried on edge)

> ```
> japscan.*#@#+js(trusted-suppress-native-method, HTMLElement.prototype.querySelectorAll, '"/.*\[[^imns].+\].*/"')
> japscan.*#@#+js(trusted-suppress-native-method, Element.prototype.hasAttribute, '"/[\S\s]+/"', prevent, inlineScript)
> japscan.*#@#+js(trusted-suppress-native-method, Document.prototype.getElementsByTagName, '"*"', prevent, inlineScript)
> ```

the issue persist on firefox without those rules.

**Comment by Yuki2718** [comment]
The custome rules won't be needed any longer.

**Comment by Ever4engel** [comment]
ye i confirm, no need for those custom rules anymore.

**Comment by Ever4engel** [comment]
okey, noticed something:

Ub0 not detected on manga series as time of writing. exemple `https://www.japscan.foo/manga/hajime-no-ippo/1513/`
in webtoons Ub0 detected in reader. sample `https://www.japscan.foo/manhwa/bad-born-blood/72/`

i guess because of `/manga/` and `/manhwa/` paths.
there also `/bd/` path exemple `https://www.japscan.foo/bd/109-rue-des-soupirs/volume-6/`

there other paths:
`/manhua/` path exemple `https://www.japscan.foo/manhua/devil-summoner-i-am-the-abyss-lord/36/`
`/comic/` path exemple `https://www.japscan.foo/comic/el-borbah/volume-1/`

so to summarize:

there: `/manga/`, `/manhwa/`, `/bd/`, `/manhua/` and `/comic/`

right now, only in `/manga/` path Ub0 isn't detected (_yet!_).

**edit:** when reader is on webtoon longstip mode UB0 detected whatever is the path.

exemple `https://www.japscan.foo/manga/the-game-architect-of-the-abyss/9/`

edit2: `https://www.japscan.foo/manga/dandadan/225` this is a manga and it's detected so i dunno anymore..

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
https://github.com/user-attachments/assets/ad9a3d04-881c-4297-a962-b3b3bc996f91" />


stuck indefinitely on CF challenge (Firefox 147.0.3).

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
stuck indefinitely on CF challenge (Firefox 147.0.3).

**Comment by Yuki2718** [comment]
The behavior of CF-check is different between Chromium and Firefox, very annoying.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

side note:

they implemented this custom captcha in the reader.

https://github.com/user-attachments/assets/726b4956-8d83-41e3-9aff-5b53aa6847ea" />

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

ps: there an issue with cloudflare chalange; despite me having upstream lists it get stuck on a blank white image, and when i update quick fixes + fr easylist i refresh it works.

**Comment by Ever4engel** [comment]
the cloudflare issue is still there. (firefox 148.0.2).

and Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

there an issue with cloudflare challenge (firefox 149.0)

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

there an issue with cloudflare challenge (firefox 149.0)
i'm starting to think the cloudflare breakage for the challange is from the french easylist.

**Comment by Yuki2718** [comment]
> 
> there an issue with cloudflare challenge (firefox 149.0) i'm starting to think the cloudflare breakage for the challange is from the french easylist.

That is due to too much recursive hooking by our filters and somewhat unstable. I reduced the number of injected filters, probably it now should be fine.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

cloudflare issue fixed.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
Ub0 detected in reader.

**Comment by Ever4engel** [comment]
https://github.com/user-attachments/assets/2ef57aa3-4108-4507-8666-ea46c96d25e4" />

challenge stuck indefinitely. (Firefox 149.0.2)

**Comment by Yuki2718** [comment]
~~I can't reprorudce on my Firefox. Does that happen every time? Maybe try refreshing the page.~~ Looks like a CF issue, I can't proceed even with uBO turned off.

**Comment by Ever4engel** [comment]
when UB0 activated:

https://github.com/user-attachments/assets/f5bd41c8-a11a-4912-9be6-be5cc015a446

when i disable it everything is fine (Firefox 150.0)
